<html>
<head>
<style>
body {
	padding:1em;
	margin:1em;
	font-family: sans-serif;
	font-size: 1em;
	line-height: 1.8em;
}

h1 {
	font-size: 2em;
	line-height: 2em;
}
</style>

<link rel="stylesheet" href="pubtator.css">
<link rel="stylesheet" href="extra.css">

</head>
<body><h1>Prevalence of <span class="Gene">ESR1</span> Mutation in Chinese <span class="Gene">ER</span>-Positive Breast <span class="Disease">Cancer</span></h1><h3>Background</h3><p><span class="Gene">ESR1</span> mutation and its possible relation to endocrine therapy resistance in <span class="Gene">ER</span>-positive <span class="Disease">breast cancers</span> have been studied with respect to genetic sequencing data from Western <span class="Species">patients</span> but rarely from Chinese <span class="Species">patients</span>. This study aimed to investigate the prevalence of <span class="Gene">ESR1</span> mutation in Chinese primary and metastatic <span class="Gene">ER</span>-positive breast <span class="Disease">cancer</span>.</p><h3>Methods</h3><p>Tumor samples from 297 primary <span class="Disease">breast cancer</span> (PBC) <span class="Species">patients</span> and blood samples from 43 metastatic <span class="Disease">breast cancer</span> (<span class="Disease">MBC</span>) <span class="Species">patients</span> were obtained to perform whole exon sequencing of the <span class="Gene">ESR1</span> gene through next-generation sequencing (NGS). Clinicopathological features of <span class="Disease">MBC</span> <span class="Species">patients</span> were listed and grouped to explore potential factors in <span class="Gene">ESR1</span> mutations.</p><h3>Results</h3><p>A total of 15 <span class="Gene">ESR1</span> variations, including 11 point mutations, 1 in-frame deletion mutation, 1 synonymous mutation, and 2 amplifications were identified in 13 <span class="Species">patients</span>. The <span class="Gene">ESR1</span> mutation rate was 1% (3/297) in <span class="Chemical">PBC</span> <span class="Species">patients</span> and 18.6% (8/43) in <span class="Disease">MBC</span> <span class="Species">patients</span>. All <span class="Gene">ESR1</span> point mutations occurred in the <span class="Gene">estrogen receptor</span> ligand-binding domain. Six (54.5%) of the 11 point mutations were hotspot mutations. Among all <span class="Disease">MBC</span> <span class="Species">patients</span>, the <span class="Gene">ESR1</span> mutation rate in those who had a treatment history using <span class="Gene">aromatase</span> inhibitors (AI) was significantly higher than those who did not (25.8% versus 0%, P=0.015). Moreover, the <span class="Gene">ESR1</span> mutation rate in those who received AI treatment over a period of 12 months was significantly higher than in those whose treatment lasted less than 12 months [36.3% versus 0%, P<0.001].</p><h3>Conclusion</h3><p><span class="Gene">ESR1</span> mutations were more frequently observed in the circulating cell-free DNA of <span class="Disease">MBC</span> <span class="Species">patients</span> than in PBC <span class="Species">patients</span> among the Chinese cohort, and higher among those pretreated with AI, suggesting that such mutations may undergo selection during AI treatment.</p><h2>Introduction</h2><p><span class="Disease">Breast cancer</span> accounted for 15.1% of all newly diagnosed <span class="Disease">cancers</span> in the Chinese <span class="Species">woman</span> cohort from 2009 to 2011. Endocrine therapy targeting <span class="Gene">estrogen receptors</span> (<span class="Gene">ER</span>) can significantly lower the risk of relapse in the early stages of <span class="Disease">breast cancer</span> as well as improve the survival outcomes for <span class="Species">patients</span> with advanced <span class="Disease">breast cancer</span>. However, it is reported that a certain number of <span class="Disease">endocrine-resistant breast cancers</span> could occur after endocrine therapy, resulting in <span class="Disease">cancer</span> recurrence or metastasis. Endocrine therapy is widely used to treat <span class="Gene">ER</span>-positive <span class="Disease">breast cancers</span> in the Chinese <span class="Species">patient</span> cohort, although the <span class="Gene">ER</span>-positive rate in Chinese <span class="Disease">breast cancer</span> <span class="Species">patients</span> is lower than in Western <span class="Species">patient</span> (50-60% vs 70%). Nevertheless, this therapy is used while knowing little about endocrine therapy resistance in the Chinese <span class="Species">patient</span> cohort.</p><p>There are multiple reasons for endocrine therapy resistance, such as loss of <span class="Gene">ERalpha</span>, up-regulation of <span class="Gene">ERbeta</span> and cross-talk between ER-genomic and growth factor pathways. As a ligand-activated nuclear hormone receptor, the <span class="Gene">ERalpha</span> can regulate cell growth, thus affect survival and metastasis in most <span class="Disease">breast cancers</span>. Many studies have found that variations of the <span class="Gene">ERalpha</span> encoding gene <span class="Gene">ESR1</span> might be a critical factor leading to endocrine therapy resistance. Among different types of <span class="Gene">ESR1</span> genomic variations, point missense mutations in <span class="Gene">ERalpha</span> ligand-binding domains (LBD) have been most commonly observed. <span class="Gene">ESR1</span> mutations may induce ligand-independent activation of <span class="Gene">ERalpha</span>, which further leads to a conformational change of <span class="Gene">ER</span> and possible endocrine therapy resistance of ER-positive <span class="Disease">breast cancers</span>. However, those findings were mostly based upon Western <span class="Species">patients</span>. The <span class="Gene">ESR1</span> mutation situation in Chinese <span class="Disease">breast cancer</span> <span class="Species">patients</span> remains unclear. Caution should be taken when using these <span class="Gene">ESR1</span> mutation data to assist treatment decisions concerning the utilization of endocrine therapy with Chinese <span class="Disease">breast cancer</span> <span class="Species">patients</span>.</p><p>The identification of <span class="Gene">ESR1</span> mutation is affected by the genetic detection method. Although digital PCR is commonly used to identify known <span class="Gene">ESR1</span> mutations of circulating <span class="Disease">tumor</span> DNA (ctDNA), the next-generation sequencing (NGS) assay can enable higher throughput and more comprehensive gene sequencing. NGS has been used to detect novel <span class="Gene">ESR1</span> mutations in ctDNA with a reported error rate as low as 0.01%. Therefore, the purpose of this study was to perform NGS assay on all exons of the <span class="Gene">ESR1</span> gene taken from primary <span class="Disease">tumor</span> samples and blood samples of metastatic <span class="Disease">breast cancer</span> to identify the prevalence of <span class="Gene">ESR1</span> mutation in Chinese <span class="Disease">breast cancer</span> <span class="Species">patients</span>.</p><h2>Materials and Methods</h2><h3><span class="Species">Patients</span> and Samples</h3><p>Pathological Characteristics of <span class="Species">Patients</span> Enrolled</p><p>Characteristics <span class="Species">Patient</span> Counts   Age (Year)     >=60 62    40-60 195    <=40 83   Menstrual Status     Pre-menopause 199    Post-menopause 141   Pathological <span class="Disease">Type     Invasive ductal carcinoma 301    Invasive lobular carcinoma</span> 11    Others 28   Pathological Grade     I 30    II 181    III 129   <span class="Gene">PR</span> Status     >=1% 314    <1% 26   <span class="Gene">HER2</span> Status     Negative 244    Positive 73    Unknown 23   Ki67     >=14 206    <14 134   TNM Stage     I 90    II 140    IIIA 37    IIIB 12    IIIC 18    IV 43   Endocrine Therapy History     Yes 41    No 299   </p><p>A total of 340 <span class="Species">patients</span>, including 297 <span class="Disease">primary breast cancer</span> (PBC) <span class="Species">patients</span> and 43 metastatic <span class="Disease">breast cancer</span> (<span class="Disease">MBC</span>) <span class="Species">patients</span>, were recruited for this study. Tumor tissue samples were collected from PBC <span class="Species">patients</span>, while 10 mL peripheral blood samples were collected from <span class="Disease">MBC</span> <span class="Species">patients</span> and stored in <span class="Disease">Struck tubes</span>. Critical pathological characteristics, such as pathological grade and <span class="Gene">progesterone receptor</span> (<span class="Gene">PR</span>) status, were determined through standard pathological tests and shown in Table 1. <span class="Gene">Human epidermal growth factor receptor-2</span> (<span class="Gene">HER2</span>) status was determined by immunohistochemistry (IHC) test. If an equivocal IHC result was found, fluorescence in situ hybridization (FISH) test was implemented to further determine <span class="Gene">HER2</span> status. Progression-free survival (PFS), defined as the time from starting treatment after <span class="Gene">ESR1</span> mutations to documented disease progression or <span class="Disease">death</span>, was analyzed based on the data cutoff in April 2018. This study was conducted in accordance with the Declaration of Helsinki and approved by the ethical review board of the Guangdong Provincial <span class="Species">People</span>'s Hospital. All <span class="Species">patients</span> signed the informed consent before the study. </p><h3>DNA Sample Extraction</h3><p>For <span class="Disease">tumor</span> tissue samples, the DNA sample was extracted using a QIAamp DNA FFPE tissue <span class="Gene">kit</span> (Qiagen, California, US). DNA concentration was measured through a Qubit dsDNA assay (Life Technologies, California, US). Blood samples were first kept at room temperature for 2 hrs before DNA extraction. The supernatant was transferred into a 15-mL centrifuge tube and then centrifuged for 10 min at 16,000 rpm at 4 C. Circulated cell-free DNA (cfDNA) was recovered from 4 to 5 mL of plasma using a QIAamp Circulating Nucleic Acid <span class="Gene">kit</span> (Qiagen, California, US). Quantification of cfDNA was performed using a Qubit 2.0 Fluorimeter (manufacturer) with a double-stranded (dsDNA) HS assay <span class="Gene">kit</span> (Life Technologies, Carlsbad, CA). At least 50ng of cfDNA was required for NGS library construction.</p><h3>NGS Library Preparation and Capture-Based Targeted DNA Sequencing</h3><p>The extracted DNA sample was subjected to end repair, phosphorylation and adaptor ligation. DNA fragments sized 200-400 bp were selected by AM-Pure beads (Agencourt AMPure XP <span class="Gene">Kit</span>), and then subjected to hybridization with capture probe baits, hybrid selection with magnetic beads and PCR amplification. A bio-analyzer high-sensitivity DNA assay was subsequently performed to assess the quality and size of the DNA fragments. Indexed samples were sequenced on Nextseq500 sequencer (Illumina, Inc., US) with pair-end reads.</p><h3>Sequence Data Analysis</h3><p>Sequence data were mapped to the <span class="Species">human</span> genome (hg19) using a BWA aligner 0.7.10. Local alignment optimization, variant calling and annotation were performed using a GATK 3.2, MuTect, and VarScan, respectively. A plasma sample was compared against its own white blood cell control sample to identify somatic variants. Variants were filtered using the VarScan fpfilter pipeline, with loci having depths less than 100 filtered out. At least 2 and 5 supporting reads were needed for insertions and deletions (InDels) in plasma and tissue samples, respectively, while 8 supporting reads were needed for SNVs to be called in both plasma and tissue samples. According to the ExAC, 1000 Genomes, dbSNP, ESP6500SI-V2 database, variants with a population frequency over 0.1% were grouped as Single Nucleotide Polymorphisms (SNP) and excluded from further analysis. Remaining variants were annotated with ANNOVAR and SnpEff v3.6. DNA translocation analysis was performed using both Tophat2 and Factera 1.4.3.</p><p>The "normalized" ctDNA abundance was calculated using two different methods. First, the maximum allele frequency of all somatic mutations detected in each plasma sample was used as the surrogate of the proportion of ctDNA among all cfDNA extracted, and the total ctDNA amount was imputed by multiplying the total ctDNA amount by that proportion. Second, for mutations that were detected in both tissue and plasma samples, the ratio of allele frequencies found in plasma to that found in tissue was calculated, and the relative ctDNA abundance was defined as the average allele frequency ratio within each <span class="Species">patient</span>, capped at 1.</p><h3>Statistical Analysis</h3><p>The ratio and mode of <span class="Gene">ESR1</span> variation were calculated and descriptively reported according to the NGS results. All <span class="Disease">MBC</span> <span class="Species">patients</span> were grouped according to different pathological characteristics such as prior hormonal therapy, metastasis sites, and <span class="Gene">HER2</span> status. chi2 tests were carried out to determine whether there was any difference in <span class="Gene">ESR1</span> mutation rates among the subgroups. The two-sided P values were nominal without adjustment for multiple testing. All analyses were hypothesis-driven. The significance level was set as 0.05. The clinical characteristics of <span class="Gene">ESR1</span>-mutant <span class="Disease">MBC</span> <span class="Species">patients</span> were reported. Statistical analysis was performed using SPSS statistical software (SPSS 13.0; SPSS Inc., Chicago, IL, USA).</p><h2>Results</h2><p>According to the results of the NGS assay, a total of 15 <span class="Gene">ESR1</span> variations in 13 <span class="Species">patients</span> were identified, including 13 <span class="Gene">ESR1</span> point mutations (substitutions or in-deletions) and 2 <span class="Gene">ESR1</span> amplifications. Among the 297 PBC <span class="Species">patients</span>, 5 <span class="Gene">ESR1</span> variant <span class="Species">patients</span> including 3 <span class="Gene">ESR1</span> point mutant <span class="Species">patients</span> and 2 CNV <span class="Species">patients</span> were identified, resulting in a 1% (3/297) <span class="Gene">ESR1</span> mutation rate among the PBC <span class="Species">patients</span>. 8 <span class="Gene">ESR1</span> mutation <span class="Species">patients</span> were identified among the 43 <span class="Disease">MBC</span> <span class="Species">patients</span>, resulting in an 18.6% (8/43) <span class="Gene">ESR1</span> mutation rate in the <span class="Disease">MBC</span> <span class="Species">patients</span>.</p><p>The 3 <span class="Gene">ESR1</span> mutations in the PBC <span class="Species">patients</span> comprised 3 mutation types: <span class="Mutation">R555C</span>, <span class="Mutation">P535S</span>, and A571 synonymous. The 10 <span class="Gene">ESR1</span> mutations in the 8 <span class="Disease">MBC</span> <span class="Species">patients</span> comprised 6 mutations type: <span class="Mutation">D538G</span> (3/10), <span class="Mutation">V422</span> in-frame deletion (1/10), <span class="Mutation">E380Q</span> (2/10), <span class="Mutation">Y537N</span> (1/10), <span class="Mutation">Y537S</span> (2/10), and <span class="Mutation">L536H</span> (1/10). All 13 <span class="Gene">ESR1</span> mutations were located within ER LBD, and 6 were hotspot mutations. The <span class="Gene">ESR1</span> mutations <span class="Mutation">D538G</span>, <span class="Mutation">E380Q</span>, and <span class="Mutation">Y537S</span> were recurrent mutations, which were all in the <span class="Disease">MBC</span> <span class="Species">patients</span>. In addition, 2 <span class="Disease">MBC</span> <span class="Species">patients</span> exhibited more than one <span class="Gene">ESR1</span> mutation (<span class="Mutation">D538G</span> plus <span class="Mutation">V422 del</span>, <span class="Mutation">E380Q</span> plus <span class="Mutation">Y537N</span>). Finally, for 3 <span class="Gene">ESR1</span>-mutant <span class="Disease">MBC</span> <span class="Species">patients</span> who could be matched to their pretreatment <span class="Disease">breast tumor</span> tissues, no <span class="Gene">ESR1</span> mutation was detected in the earlier samples.</p><p>Difference of <span class="Gene">ESR1</span> Mutation Rate in Metastatic <span class="Disease">Breast Cancer</span> <span class="Species">Patients</span> Grouped by Pathological Characteristics</p><p>Characteristics <span class="Gene">ESR1</span>-Mutant <span class="Species">Patients</span> <span class="Gene">ESR1</span> Wild-Type <span class="Species">Patients</span> P   <span class="Gene">HER2</span> Status       Positive 1(12.5%) 7(87.5%) 0.561    Negative 7(20%) 26(80%)   Ki67       >=14 7(18.9%) 30(81.1%) 0.894    <14 1(16.7%) 5(83.3%)   Menstrual Status       Pre-menopause 3(13%) 20(87%) 0.314    Post-menopause 5(25%) 15(75%)   Disease Metastasis Sites       1 4(13.3%) 26(86.7%) 0.191    >=2 4(30.8%) 9(69.2%)   Visceral Metastasis       Yes 6(28.6%) 15(71.4%) 0.095    No 2(9%) 20(91%)   Bone Metastasis       Yes 4(19%) 17(81%) 0.942    No 4(18.2%) 18(81.8%)   Endocrine Therapy History       Yes 8(19.5%) 33(80.5%) 0.358    No 0(0%) 2(100%)   Previous Endocrine Therapy Courses       <=1 4(23.5%) 13(76.5%) 0.506    >=2 4(15.4%) 22(84.6%)   AIs Treatment History       Yes 8 (25.8%) 23 (74.2%) 0.015*    No 0 (0%) 12 (100%)   AIs Treatment Time (Month)       <12 0 (0%) 21 (100%) <0.001*    >=12 8 (36.3%) 14 (63.7%)   </p><p>Notes: *Significant difference between each subgroup of the clinicopathological characteristics.</p><p>The differences between <span class="Gene">ESR1</span> mutation rates of subgroups of clinicopathological characteristics are shown in Table 2. <span class="Gene">ESR1</span> mutations were found to be significantly affected by AI treatment. The <span class="Gene">ESR1</span> mutation rate was significantly higher in <span class="Disease">MBC</span> <span class="Species">patients</span> who had a history of AI treatment than in those who did not (P=0.015). In addition, <span class="Disease">MBC</span> <span class="Species">patients</span> who had AI treatment for over 12 months had significantly higher <span class="Gene">ESR1</span> mutation rates than those whose treatment was less than 12 months (P<0.001). No significant difference in <span class="Gene">ESR1</span> mutation rates was observed between different <span class="Gene">HER2</span> status, menstrual status, metastasis sites, and previous endocrine exposure. </p><p>Clinical Features of <span class="Gene">ER</span>-Positive <span class="Gene">ESR1</span>-Mutant Metastatic <span class="Species">Patients</span> (N=8)</p><p><span class="Species">Patients</span> DNA Alteration Code and Alteration Frequency Amino Changes <span class="Gene">HER2</span> Status Pathologic Type and Grade Metastasized Organs Endocrine Therapy Drugs AIs Therapy Time (Month)   Case 1 <span class="Mutation">1138G>C</span> 2.72%<span class="Mutation">1609T>A</span> 0.69% E380QY537N Positive Invasive <span class="Disease">ductal carcinoma</span>, grade II Bone, lung, liver, soft tissues Letrozole 25   Case 2 <span class="Mutation">1613A>G</span> 2.74%1265_<span class="Mutation">1267del</span> 0.47% <span class="Mutation">D538GV422del</span> Negative <span class="Disease">Invasive ductal carcinoma</span>, grade III Bone Exemestane 28   Case 3 <span class="Mutation">1610A>C</span> 2.07% <span class="Mutation">Y537S</span> Negative Invasive micro-<span class="Disease">papillary carcinoma</span>, grade unknown Lung Anastrozole 18   Case 4 <span class="Mutation">1610A>C</span> 1.09% <span class="Mutation">Y537S</span> Negative <span class="Disease">Invasive mucinous carcinoma</span>, grade I Bone, lung, liver Letrozole 22   Case 5 <span class="Mutation">1138G>C</span> 4.26% <span class="Mutation">E380Q</span> Negative <span class="Disease">Invasive ductal carcinoma</span>, grade II Liver Anastrozole 16   Case 6 <span class="Mutation">1613A>G</span> 2.83% <span class="Mutation">D538G</span> Negative <span class="Disease">Invasive ductal carcinoma</span>, grade III Bone, lung, liver Exemestane 48   Case 7 <span class="Mutation">1607_1608delinsAT</span> 9.3% <span class="Mutation">L536H</span> Negative <span class="Disease">Invasive ductal carcinoma</span>, grade III Chest wall Letrozole 29   Case 8 <span class="Mutation">1613A>G</span> 6.6% <span class="Mutation">D538G</span> Negative <span class="Disease">Invasive ductal carcinoma</span>, grade III Lung, liver, soft tissue Tamoxifenanastrozole 16   </p><p>The endocrine therapy histories for the 8 <span class="Gene">ESR1</span>-mutant <span class="Species">patients</span> were retrospectively reviewed (Table 3). The 8 <span class="Gene">ESR1</span>-mutant <span class="Species">patients</span> had an average AI treatment period of 25.2 months. The therapies of 5 <span class="Species">patients</span> were subsequently changed to stimulating <span class="Gene">estrogen receptor</span> degradation drugs (SERDs) such as fulvestrant (Figures S11-8). All of them exhibited long survival time with stable disease. The other 3 <span class="Species">patients</span> <span class="Disease">died</span> without further systemic therapy, with the longest survival time of 4 months (Figures S11-8). </p><h2>Discussion</h2><p>Although <span class="Gene">ESR1</span> mutations have long been perceived to play a role in the endocrine therapy resistance of <span class="Gene">ER</span>-positive <span class="Disease">breast cancers</span>, <span class="Gene">ESR1</span> mutations in Chinese <span class="Species">patients</span> have been rarely studied. This is the first study investigating the prevalence of <span class="Gene">ESR1</span> mutation in <span class="Gene">ER</span>-positive <span class="Disease">breast cancers</span> in Chinese cohorts. In the present study, whole exon sequencing of <span class="Gene">ESR1</span> with NGS was performed on <span class="Disease">tumors</span> or blood samples of 340 <span class="Gene">ER</span>-positive PBC and <span class="Disease">MBC</span> <span class="Species">patients</span>. <span class="Gene">ESR1</span> mutations were detected in 3.2% (11/340) of all enrolled <span class="Species">patients</span>. Specially, the <span class="Gene">ESR1</span> mutation rate was only 1% in the <span class="Chemical">PBC</span> <span class="Species">patients</span>, which was consistent with previous studies reporting that <span class="Gene">ESR1</span> mutations were seldom detected in Western PBC <span class="Species">patients</span>. This rate in the present study was slightly lower than the reported ER LBD mutation of 3% in the primary <span class="Disease">tumors</span> (<span class="Disease">tumor</span> without exposure to hormonal therapy) in the <span class="Chemical">BOLERO-2</span> clinical trial. However, Jeselsohn and co-workers found no ER LBD mutation among 58 primary <span class="Disease">tumors</span>. In this present study, the <span class="Gene">ESR1</span> mutation rate increased to 18.6% in the <span class="Disease">MBC</span> <span class="Species">patients</span>, which was within the rate of 14-54% reported by previous studies based on Western <span class="Disease">MBC</span> <span class="Species">patients</span>. Moreover, 3 <span class="Gene">ESR1</span>-mutant <span class="Disease">MBC</span> <span class="Species">patients</span> could be matched to their pretreatment <span class="Disease">breast tumor</span> tissues where no <span class="Gene">ESR1</span> mutation was detected. The higher <span class="Gene">ESR1</span> mutation rate in the <span class="Disease">MBC</span> <span class="Species">patients</span> indicates that <span class="Gene">ESR1</span> mutation might be associated with <span class="Disease">breast cancer</span> progression. According to previous studies, the majority of <span class="Gene">ESR1</span> mutations were located within ER LBD, particularly hotspot mutations in residues <span class="Mutation">Y537</span> and <span class="Mutation">D538</span>. In our study, all 13 <span class="Gene">ESR1</span> mutations were located within ER LBD, including 6 hotspot mutations. In addition, we detected two <span class="Gene">ESR1</span> amplifications in <span class="Chemical">PBC</span> <span class="Species">patients</span> but none in <span class="Disease">MBC</span> <span class="Species">patients</span>, with a very low <span class="Gene">ESR1</span> amplification rate. This is close to the previously reported <span class="Gene">ESR1</span> amplification rate of 1.5-6% in <span class="Disease">breast cancer</span> <span class="Species">patients</span>. A higher frequency of <span class="Gene">ESR1</span> amplification has been reported in PBC <span class="Species">patients</span>, which may be influenced by the detection method and scoring system.</p><p>In previous studies, most <span class="Species">patients</span> received AI treatment prior to the detection of <span class="Gene">ESR1</span> mutation, which makes it necessary to explore the possible relation between the possible relation between <span class="Gene">ESR1</span> mutation and AI treatment. In the present study, <span class="Gene">ESR1</span> mutations were found to be associated with AI treatment. An <span class="Gene">ESR1</span> mutation rate of 25.8% was identified in the <span class="Disease">MBC</span> <span class="Species">patients</span> who had a history of AI treatment, which was close to the average rate of 23% reported in previous studies. Meanwhile, the <span class="Gene">ESR1</span> mutation rate was 0% in <span class="Disease">MBC</span> <span class="Species">patients</span> without a history of AI treatment. In addition, the <span class="Gene">ESR1</span> mutation rate was higher in <span class="Disease">MBC</span> <span class="Species">patients</span> who had AI treatment for over 12 months than in those whose treatment was shorter than 12 months. The possible mechanism behind this may be that <span class="Gene">ESR1</span>-mutant clones become prominent and harbor a growth advantage over other cells under the estrogen-deprivation caused by successive lines of AI treatment. Previous studies showed that <span class="Gene">ESR1</span> mutations were less frequently observed in <span class="Species">patients</span> who received AI treatment only at an adjuvant setting, suggesting the clonal selection of <span class="Gene">ESR1</span>-mutant <span class="Disease">cancer</span> cells under the estrogen-deprived condition in the metastatic <span class="Disease">tumors</span>, but not in the small number of <span class="Disease">tumors</span> during adjuvant treatment.</p><p>A previous study reported that AI resistance might be associated with <span class="Gene">ESR1</span> mutation, and therefore <span class="Gene">ESR1</span>-mutant <span class="Species">patients</span> should avoid subsequent AI treatment. Another study also found that the overall survival time of <span class="Gene">ESR1</span> mutation <span class="Species">patients</span> (i.e., <span class="Mutation">D538G</span>, 25.99 months; <span class="Mutation">Y537S</span>, 19.98 months; both <span class="Mutation">D538G</span> and <span class="Mutation">Y537S</span> mutations, 15.15 months) was shorter than <span class="Gene">ESR1</span> wild-type <span class="Species">patients</span> (32.1 months). For the 8 <span class="Gene">ESR1</span>-mutant <span class="Disease">MBC</span> <span class="Species">patients</span> in this study, the therapies of 5 <span class="Species">patients</span> were changed to <span class="Chemical">SERDs</span> such as fulvestrant. They exhibited long survival time with stable disease, indicating substantial benefits from this therapy change.</p><p>Several limitations in this study should be clarified. The present study had a relatively smaller sample size, which could be one of the reasons why several <span class="Gene">ESR1</span> mutation types (i.e., <span class="Mutation">K303R</span> and <span class="Mutation">S463P</span>) were not detected. Moreover, it prevented us from determining whether <span class="Gene">ESR1</span> mutation detection could predict sensitivity to specific hormone therapies or combinations, such as fulvestrant and/or <span class="Gene">CDK4/6</span> inhibitors. More studies should be performed to address whether there are clinical differences between <span class="Gene">ESR1</span> mutations, and to assess the lead time between the emergence of <span class="Gene">ESR1</span> mutations and the clinical progression of the disease through longitudinal sampling. In this study, we performed liquid biopsy rather than <span class="Disease">tumor</span> biopsy in the <span class="Disease">MBC</span> <span class="Species">patient</span>, in order to minimize the effect of inter-<span class="Disease">tumor</span> heterogeneity on the accuracy of <span class="Gene">ESR1</span> mutation examination. To address the potential effect of sample selection, further study should focus on studying the concordance of <span class="Gene">ESR1</span> mutation pattern in <span class="Disease">MBC</span> <span class="Species">patients</span> between different biopsies.</p><h2>Conclusions</h2><p>As the first study using NGS to identify <span class="Gene">ESR1</span> mutations in <span class="Gene">ER</span>-positive breast <span class="Disease">cancer</span> based on Chinese cohorts, our study showed that <span class="Gene">ESR1</span> mutations are rare in untreated primary <span class="Disease">tumors</span> but are more likely to occur in metastatic <span class="Species">patients</span> with a history of AI treatment, indicating that such mutations may undergo selection during AI treatment. Future studies should identify molecular mechanisms of <span class="Gene">ESR1</span> mutations in AI resistance during the progression of <span class="Gene">ER</span>-positive breast <span class="Disease">cancer</span>.</p><h2>Funding</h2><p>This work was supported by funding from the National Natural Science Foundation of China (81602645; 81071851; 81001189), Natural Science Foundation of Guangdong Province (2016A030313768; 2018A030313292) and Research Funds from Guangzhou Municipal Science and Technology Project (201707010418; 201804010430).</p><h2>Disclosure</h2><p>The authors report no conflicts of interest in this work.</p><h2>References</h2><p><span class="Disease">Cancer</span> statistics in China, 2015</p><p><span class="Gene">Aromatase</span> inhibitors in <span class="Disease">breast cancer</span></p><p><span class="Gene">Estrogen receptor alpha</span> somatic mutations <span class="Mutation">Y537S</span> and <span class="Mutation">D538G</span> confer <span class="Disease">breast cancer</span> endocrine resistance by stabilizing the activating function-2 binding conformation</p><p>Effects of chemotherapy and hormonal therapy for early <span class="Disease">breast cancer</span> on recurrence and 15-year survival: an overview of the randomised trials</p><p>Biological determinants of endocrine resistance in <span class="Disease">breast cancer</span></p><p><span class="Disease">Breast cancer</span> in China</p><p>Quantitative <span class="Gene">estrogen receptor</span> analyses: the response to endocrine and cytotoxic chemotherapy in <span class="Species">human</span> <span class="Disease">breast cancer</span> and the disease-free interval</p><p>Coexpression of <span class="Gene">estrogen receptor alpha</span> and beta: poor prognostic factors in <span class="Species">human</span> <span class="Disease">breast cancer</span>?</p><p>Growth factor receptor cross-talk with estrogen receptor as a mechanism for <span class="Chemical">tamoxifen</span> resistance in <span class="Disease">breast cancer</span></p><p>Endocrine-therapy-resistant <span class="Gene">ESR1</span> variants revealed by genomic characterization of <span class="Disease">breast-cancer</span>-derived xenografts</p><p><span class="Gene">ESR1</span> ligand-binding domain mutations in hormone-resistant <span class="Disease">breast cancer</span></p><p>Activating <span class="Gene">ESR1</span> mutations in hormone-resistant metastatic <span class="Disease">breast cancer</span></p><p><span class="Gene">ESR1</span> mutations-a mechanism for acquired endocrine resistance in <span class="Disease">breast cancer</span></p><p><span class="Gene">ESR1</span> and <span class="Gene">EGF</span> genetic variation in relation to <span class="Disease">breast cancer</span> risk and survival</p><p>Emergence of constitutively active <span class="Gene">estrogen receptor-alpha</span> mutations in pretreated advanced <span class="Gene">estrogen receptor</span>-positive breast <span class="Disease">cancer</span></p><p><span class="Mutation">D538G</span> mutation in <span class="Gene">estrogen receptor-alpha</span>: a novel mechanism for acquired endocrine resistance in <span class="Disease">breast cancer</span></p><p>Analysis of <span class="Gene">ESR1</span> mutation in circulating <span class="Disease">tumor</span> DNA demonstrates evolution during therapy for metastatic <span class="Disease">breast cancer</span></p><p>Sensitive detection of mono- and polyclonal <span class="Gene">ESR1</span> mutations in primary <span class="Disease">tumors</span>, metastatic lesions, and cell-free DNA of <span class="Disease">breast cancer</span> patients</p><p>Prevalence of <span class="Gene">ESR1</span> mutations in cell-free DNA and outcomes in metastatic <span class="Disease">breast cancer</span>: a secondary analysis of the BOLERO-2 clinical trial</p><p>High-fidelity target sequencing of individual molecules identified using barcode sequences: de novo detection and absolute quantitation of mutations in plasma cell-free DNA from <span class="Disease">cancer</span> patients</p><p>Field guide to next-generation DNA sequencers</p><p>Activating <span class="Gene">ESR1</span> mutations differentially affect the efficacy of <span class="Gene">ER</span> antagonists</p><p>Detection of <span class="Gene">ESR1</span> mutations in plasma and <span class="Disease">tumors</span> from metastatic <span class="Disease">breast cancer</span> <span class="Species">patients</span> using next-generation sequencing</p><p>American Society of Clinical Oncology/College Of American Pathologists guideline recommendations for immunohistochemical testing of estrogen and progesterone receptors in <span class="Disease">breast cancer</span></p><p>Recommendations for <span class="Species">human</span> <span class="Gene">epidermal growth factor receptor 2</span> testing in <span class="Disease">breast cancer</span>: American Society of Clinical Oncology/College of American Pathologists clinical practice guideline update</p><p>Tracking plasma DNA mutation dynamics in <span class="Gene">estrogen receptor</span> positive metastatic <span class="Disease">breast cancer</span> with dPCR-SEQ</p><p>Comparison of <span class="Gene">ESR1</span> mutations in <span class="Disease">tumor</span> tissue and matched plasma samples from metastatic <span class="Disease">breast cancer</span> patients</p><p>Comprehensive molecular portraits of <span class="Species">human</span> <span class="Disease">breast tumours</span></p><p>Absence of <span class="Gene">ESR1</span> amplification in a series of <span class="Disease">breast cancers</span></p><p><span class="Gene">Estrogen receptor alpha</span> (<span class="Gene">ESR1</span>) gene amplification is frequent in <span class="Disease">breast cancer</span></p><p>Incidence and clinical significance of <span class="Gene">ESR1</span> mutations in heavily pretreated metastatic <span class="Disease">breast cancer</span> patients</p><p>Capturing intra-<span class="Disease">tumor</span> genetic heterogeneity by de novo mutation profiling of circulating cell-free <span class="Disease">tumor</span> DNA: a proof-of-principle</p></body>
</html>